apis/v1alpha1/shared_types.go

```diff
@@ -30,12 +30,12 @@ type AdminNetworkPolicySubject struct {
 	Namespaces *metav1.LabelSelector `json:"namespaces,omitempty"`
 	// Pods is used to select pods via namespace AND pod selectors.
 	// +optional
-	Pods *NamespacedPodSubject `json:"pods,omitempty"`
+	Pods *NamespacedPod `json:"pods,omitempty"`
```

Comment:
(The removal of SameLabels/NotSameLabels means that the `subject` pod selector and the peer pod selector are now identical so the separate `NamespacedPodSubject` type was removed.)